### DISCUSSION TITLE
After upgrading Flsun V400 to Vanilla Klipper Logitech C920 camera runs at 1 FPS

### DISCUSSION THREAD

**Discussion by AbitNutzT00** [Guilouz/Klipper-Flsun-Speeder-Pad#167] [Q&A]
I wiped out the Speederpad and moved to full Klipper and am really happy with everything but the camera. On Flsun's 1.41 Speederpad the camera ran fine. After moving to full Klipper the C920 will only put out 1 FPS. I changed resolution with no luck. I tried the few flags I cold find. Here's my Crowsnest config

[crowsnest
log_path: /home/pi/printer_data/logs/crowsnest.log
log_level: verbose                      # Valid Options are quiet/verbose/debug
delete_log: false                       # Deletes log on every restart, if set to true
no_proxy: false

[cam 1]
mode: ustreamer                         # ustreamer - Provides mjpg and snapshots. (All devices)
                                        # camera-streamer - Provides webrtc, mjpg and snapshots. (rpi + Raspi OS based only)
enable_rtsp: true                    # If camera-streamer is used, this enables also usage of an rtsp server
rtsp_port: 8554                         # Set different ports for each device!
port: 8080                              # HTTP/MJPG Stream/Snapshot Port
device: /dev/video0                     # See Log for available ...
resolution: 1280x720                     # widthxheight format
max_fps: 15                             # If Hardware Supports this it will be forced, otherwise ignored/coerced.
#custom_flags: --format=YUYV                     # You can run the Stream Services with custom flags.
v4l2ctl:

**Comment by SckyzO** [comment]
Hello, 

To solve your issue, just add `custom_flags: --format=YUYV`

**Reply by AbitNutzT00** [reply]
That was it...thank you very much. I had thought I'd tried that but apparently not.
